v2/schema/context.jsonld

```diff
@@ -0,0 +1,112 @@
+{
+  "@context": {
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "@vocab": "http://www.w3.org/ns/dcat#",
+    "dc": "http://purl.org/dc/terms/",
+    "dctypes": "http://purl.org/dc/dcmitype/",
+    "gbl": "http://geoblacklight.org/v2/schema#",
+    "ogc": "http://www.opengis.net/ogc#",
+    "pod": "https://project-open-data.cio.gov/v1.1/schema#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "accrualPeriodicity": {
+      "@type": "@id",
+      "@id": "dc:accrualPeriodicity"
+    },
+    "accessLevel": {
+      "@type": "@id",
+      "@id": "pod:accessLevel"
+    },
+    "accessURL": {
+      "@type": "@id",
+      "@id": "dcat:accessURL"
+    },
+    "conformsTo": {
+      "@type": "@id",
+      "@id": "dc:conformsTo"
+    },
+    "creator": {
+      "@id": "dc:creator"
+    },
+    "describedBy": {
+      "@id": "http://www.w3.org/2007/05/powder#describedby",
+      "@type": "@id"
+    },
+    "describedByType": {
+      "@type": "@id",
+      "@id": "pod:describedByType"
+    },
+    "description": {
+      "@id": "dc:description"
+    },
+    "downloadURL": {
+      "@type": "@id",
+      "@id": "dcat:downloadURL"
+    },
+    "format": {
+      "@type": "gbl:format",
+      "@id": "dctypes:format"
+    },
+    "geom": {
+      "@id": "ogc:envelope"
```

Comment:
I'm not sure if `ogc:envelope` is a real element. Maybe something more like `http://www.opengis.net/csw/3.0/cql#envelope` or ` http://www.opengis.net/doc/IS/cat/3.0/cql#envelope`